test/appsignal/config_test.exs

```diff
@@ -88,21 +86,14 @@ defmodule Appsignal.ConfigTest do
   describe "active?" do
     test "when active and valid" do
       assert with_config(
-               %{active: true, valid: true},
+               %{active: true},
```

Comment:
Should this test still be named "when active and valid"?

Reply:
Nope, changing it